jenkins/edx_platform_test_notifier.py

```diff
@@ -1,12 +1,13 @@
+from __future__ import absolute_import
+
 import logging
-import os
```

Comment:
^ I notice you removed this as an import but not from the code. Was this just an unused import?

Reply:
Yes, PyCharm caught that.